drivers/hmis_external_apis/app/assets/javascripts/external_forms.js

```diff
@@ -221,7 +221,7 @@ window.addDependentGroup = function (conditions, targetSelector, enableBehavior
   // When *any* dependent item changes, this function will check all the conditions, and show/hide the target item accordingly.
   var onDependentItemChanged = function() {
     var evaluations = conditions.map(function (condition) {
-      var $el = $('[name="' + condition.input_name + '"]')
+      var $el = $('[name="' + condition.input_name + '"]:visible')
```

Comment:
I _think_ this is a bug due to the hidden input here: https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/drivers/hmis_external_apis/app/views/hmis_external_apis/external_forms/form/_checkbox.haml#L1 

I'm not sure if there were any existing places in our external forms where we have enableWhen conditions based on checkboxes.

Reply:
Yes just confirming there were no usages before, thanks!